### ISSUE TITLE
Add DeepAlpha - AI crypto trading bot

### ISSUE THREAD

**Issue by stefanoviana** [merovinh/best-of-algorithmic-trading#175]
Hi! I would like to suggest adding [DeepAlpha](https://github.com/stefanoviana/deepalpha) to the best-of list.

**What is DeepAlpha?**
AI crypto trading bot with 70.9% walk-forward validated accuracy. XGBoost + LightGBM ensemble with 72 features. MIT license. Available on PyPI (`pip install deepalpha`).

**Suggested projects.yaml entry:**
```yaml
- name: DeepAlpha
  github_id: stefanoviana/deepalpha
  pypi_id: deepalpha
  category: bots
  description: AI crypto trading bot with 70.9% walk-forward validated accuracy. XGBoost + LightGBM ensemble.
```

Thank you!
